### PR TITLE
Operator utils  ubi8.5

### PR DESCRIPTION
**Pull request by sapnashukla28**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by sapnashukla28** [comment]
Following script has been created by Apurva, raising PR on her behalf.

Thanks,
Sapna

File: o/operator_utils/operator_utils_ubi8.5.sh [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+PACKAGE_PATH=github.com/redhat-cop/
+PACKAGE_NAME=operator-utils
+PACKAGE_VERSION=v1.1.1
+PACKAGE_URL=https://github.com/redhat-cop/operator-utils
+
+# Install dependencies
+yum install -y make git wget gcc
+
+# Download and install go
+wget https://golang.org/dl/go1.14.13.linux-ppc64le.tar.gz
```
Comment (seth-priya) [inline o/operator_utils/operator_utils_ubi8.5.sh:33]: is there are a specific need for this go version as opposed to the latest or the distro default one?

**Comment by seth-priya** [comment]
Not a major issue, but for consistency let's name build scripts as <package_name>_<package_version>_<distro_name>_<distro_version>.sh where package_version is optional and can be used only when a build script has been created specifically for an older version or in general isn't expected to work with the top-of-tree

File: o/operator_utils/operator_utils_ubi8.5.sh [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+CWD=`pwd`
+
+PACKAGE_PATH=github.com/redhat-cop/
+PACKAGE_NAME=operator-utils
+PACKAGE_VERSION=v1.1.1
+PACKAGE_URL=https://github.com/redhat-cop/operator-utils
+
+# Install dependencies
+yum install -y make git wget gcc
```
Comment (seth-priya) [inline o/operator_utils/operator_utils_ubi8.5.sh:30]: do we need gcc?

Comment (sapnashukla28) [reply o/operator_utils/operator_utils_ubi8.5.sh]: Yes without gcc gives following error:
/home/go/pkg/tool/linux_ppc64le/link: running gcc failed: exec: "gcc": executable file not found in $PATH

File: o/operator_utils/operator_utils_ubi8.5.sh (hunk 1 above)
Comment (sapnashukla28) [reply o/operator_utils/operator_utils_ubi8.5.sh]: Changed this to 17.5

**Review by seth-priya — APPROVED** [review]

**Pull request merged**